reference/library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -110,7 +110,7 @@
 @mixin sakai_button_color( $button-color-background-color, $button-color-border-color, $button-color-text-color, $button-color-background-secondary-color ){
 	background: $button-color-background-color;
 	border: 1px solid $button-color-border-color;
-	color: $button-color-text-color !important;
```

Comment:
This ugly `!important` is needed because a lot of tools overwrite this. If there are problems with it (I understand this is an ugly solution) we can add an extra parameter to the function with the important or not if needed (maybe default false, maybe true). What do you think?